test/utils/setup.ts

```diff
@@ -7,6 +7,16 @@ import { safeContractUnderTest } from "./config";
 import { getRandomIntAsString } from "./numbers";
 import { Safe, SafeL2, SafeMigration } from "../../typechain-types";
 
+type safeWithFullConfig = {
```

Comment:
By TS naming convention, types names use `UpperCamelCase`: https://google.github.io/styleguide/tsguide.html#naming-rules-by-identifier-type

(this is google styleguide but this particular naming convention is universal)

Reply:
In particular, we use this naming convention in our project as well.

Reply:
Done 👍🏾